plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -517,6 +532,25 @@ private class DokkaDescriptorVisitor( // TODO: close this class and make it priv
         )
     }.let(::Annotations)
 
+    fun DeclarationDescriptor.convertType(dri: DRI, context: DokkaContext): Bound = when (val desc = this) {
+        is ValueParameterDescriptor -> desc.type
+        is FunctionDescriptor -> desc.returnType
+        is PropertyDescriptor -> desc.returnType
+        is ReceiverParameterDescriptor -> desc.type
+        else -> null
+    }?.let { context.unresolvedTypeHandler(it) }.let { OtherParameter(it ?: "") }
+
+    inline fun <reified T: DeclarationDescriptor> Collection<T>.filterTypeError() =
+        if (context.unresolvedTypeHandler == UnresolvedTypeHandler.Skip) {
```

Comment:
If you were intending to have the logic closed inside the particular entry, why are you letting it out now?